tests/unit_tests/document_loader/test_evernote_loader.py

```diff
@@ -0,0 +1,75 @@
+import pathlib
+import os
+import time
+
+from pytest_mock import MockerFixture
+from langchain.document_loaders import EverNoteLoader
+
+
+class TestEverNoteLoader:
+
+    @staticmethod
+    def example_notebook_path(notebook_name: str) -> str:
+        current_dir = pathlib.Path(__file__).parent
+        return os.path.join(current_dir, "sample_documents", notebook_name)
+
+    def test_evernoteloader_loadnotebook_eachnoteisindividualdocument(self) -> None:
```

Comment:
@MikeMcGarry you can use the following (still undocumented) custom marker to get the tests to run. They'll need to be registered in the `extended_testing` extra

```suggestion
    @pytest.mark.requires("lxml", "html2txt")
    def test_evernoteloader_loadnotebook_eachnoteisindividualdocument(self) -> None:
```

Reply:
Yeah sure thing, however when I add this decorator and call `make test` (after calling `poetry lock --no-update` and `poetry install -E extended_testing` all the tests are skipped?

Reply:
Ahh never mind, I see how it works, had a type in `html2txt`. Fixed this. I see that it will skip the tests if the dependency is not installed. Nice!